sdk/Cargo.toml

```diff
@@ -608,11 +603,6 @@ name = "storage"
 path = "examples/wallet/storage.rs"
 required-features = [ "wallet", "storage" ]
 
-[[example]]
-name = "threads"
-path = "examples/wallet/threads.rs"
```

Comment:
This was my good spammer example :sob:

Reply:
Why is it removed anyway?

Reply:
there are no good spammers :trollface: 
seriously tho ... can you check with the `multithread` example @Thoralf-M I thought they were basically the same, and `threads` wouldn't add much else anymore.

Reply:
It's only doing 4 rounds and missing the syncing when required, the threads you could just run to spam, this helped me to find some race conditions which is also harder if you have multiple accounts doing txs at the same time